src/transforms/add_tags.rs

```diff
@@ -58,7 +61,11 @@ impl Transform for AddTags {
 
             for (name, value) in &self.tags {
                 let map = tags.as_mut().unwrap(); // initialized earlier
-                map.insert(name.to_string(), value.to_string());
+                if self.r#override {
+                    map.insert(name.to_string(), value.to_string());
```

Comment:
do we want to do the same logging as we have in the `add_fields` transform?

Reply:
Probably!